### PR TITLE
fix(SUP-13171): adCover blocking VPAID controls

### PR DESCRIPTION
**Pull request by ErezBouskila**
@OrenMe Please review PR

### CODE REVIEW

**Comment by ErezBouskila** [comment]
@OrenMe I can use the getContentType() and check the string of the ad.

**Comment by OrenMe** [comment]
@ErezBouskila if this uses a valid API and official response then it’s fine

**Comment by ErezBouskila** [comment]
@OrenMe Made the change, here is the list of the getContentType() types:
https://groups.google.com/forum/#!topic/ima-sdk/TaKullfdqH4

Should we check for application/x-shockwave-flash?

**Comment by OrenMe** [comment]
@ErezBouskila I don't think this is safe either - unfortunately ad space is very fragmented and you can't be sure that ad networks or vendors return correct metadata.

**Comment by ErezBouskila** [comment]
@OrenMe It seems that ad.getContentType() is the only way to use the API to know if the add is vpaid:
https://groups.google.com/forum/#!topic/ima-sdk/bTy9l2CSHOs

**Comment by ErezBouskila** [comment]
@OrenMe Please review the latest fix.
Thanks

**Comment by OrenMe** [comment]
@yairans please review as well

File: modules/DoubleClick/resources/mw.DoubleClick.js [hunk 1]
```diff
@@ -1556,6 +1578,16 @@
                 'height': this.embedPlayer.getVideoHolder().height()
             };
         },
+        hideControlsOnVPAID: function(){
+            var hideControls = "-40px";
```
Comment (OrenMe) [inline modules/DoubleClick/resources/mw.DoubleClick.js]: what happens if we are with actual hover controls?
also, `-40px` - what if we have custom CSS?

Comment (OrenMe) [reply modules/DoubleClick/resources/mw.DoubleClick.js]: Maybe consider moving this to controlBars classes as methods and doing `this.embedPlayer.layoutManager.hideControls()` kind of a thing

Comment (ErezBouskila) [reply modules/DoubleClick/resources/mw.DoubleClick.js]: @OrenMe I checked with hover controls and it works, I can make the pixel size larger as I do not want to mess around with z-index.

Will move it to controlBars

**Comment by itaykinnrot** [comment]
Lets add a flag here just to be on the safe side

**Comment by ErezBouskila** [comment]
@itaykinnrot @yairans Please review PR.
Thanks

**Comment by ErezBouskila** [comment]
@OrenMe  @yairans  Made the final change (Hopefully), please review PR
Thanks

**Pull request merged**